Sources/SkyFloatingLabelTextField.swift

```diff
@@ -31,17 +31,17 @@ open class SkyFloatingLabelTextField: UITextField {
     // MARK: Animation timing
 
     /// The value of the title appearing duration
-    open var titleFadeInDuration:TimeInterval = 0.2
+    dynamic open var titleFadeInDuration:TimeInterval = 0.2
     /// The value of the title disappearing duration
-    open var titleFadeOutDuration:TimeInterval = 0.3
+    dynamic open var titleFadeOutDuration:TimeInterval = 0.3
```

Comment:
Annotating properties with`dynamic` is sufficient to support UIAppearance in Obj-C as well.